sentence_transformers/SentenceTransformer.py

```diff
@@ -1213,7 +1245,7 @@ def _load_auto_model(
                 "trust_remote_code": trust_remote_code,
                 "revision": revision,
                 "local_files_only": local_files_only,
-            },
+            } | (model_args or {}),
```

Comment:
The `|` operator isn't supported in Python 3.8, and I'd like to keep supporting that version until [~6 months from now](https://endoflife.date/python).

Reply:
👍 Updated to use Python 3.8 compatible code